crates/config/src/providers/remappings.rs

```diff
@@ -308,3 +320,131 @@ impl Provider for RemappingsProvider<'_> {
         Some(Config::selected_profile())
     }
 }
+
+#[cfg(test)]
```

Comment:
looks good, another approach to write remapping tests is similar with https://github.com/foundry-rs/foundry/blob/5e72c69e8414ec7b535eedb357e9b6db3e312b62/crates/forge/tests/cli/config.rs#L600